controllers/operatorpolicy_controller.go

```diff
@@ -300,16 +307,22 @@ func buildOperatorGroup(
 ) *operatorv1.OperatorGroup {
 	operatorGroup := new(operatorv1.OperatorGroup)
 
-	gvk := schema.GroupVersionKind{
-		Group:   "operators.coreos.com",
-		Version: "v1",
-		Kind:    "OperatorGroup",
+	// Create a default OperatorGroup if one wasn't specified in the policy
+	// Future work: Prevent creating multiple OperatorGroups in the same ns
+	if policy.Spec.OperatorGroup == nil {
+		operatorGroup.SetGroupVersionKind(operatorGroupGVK)
+		operatorGroup.ObjectMeta.SetName(policy.Spec.Subscription.Package + defaultOGSuffix)
```

Comment:
After talking with Justin, the name length maximum is 253 characters but the label values are 63. So I was mistaken.

I think you could possibly run into this but as long as you're logging the error, I think this is fine for now.